cmd/soroban-cli/src/commands/contract/init.rs

```diff
@@ -41,8 +41,8 @@ pub struct Cmd {
     pub with_example: Vec<String>,
 
     #[arg(
-        short,
         long,
+        short = 't',
```

Comment:
I think we should omit the short code rather than add a new one. We can always add one later if this is the wrong call.
```suggestion
```